evm/vm/forks/frontier/__init__.py

```diff
@@ -40,6 +39,14 @@
 )
 
 
+FRONTIER_PRECOMPILES = {
+    force_bytes_to_address(b'\x01'): precompiles.precompile_ecrecover,
+    force_bytes_to_address(b'\x02'): precompiles.precompile_sha256,
+    force_bytes_to_address(b'\x03'): precompiles.precompile_ripemd160,
+    force_bytes_to_address(b'\x04'): precompiles.precompile_identity,
```

Comment:
What do you think about removing the 'precompile_' suffix from the function names, to avoid the stutter when using them?